core/src/avm2/globals/flash/geom/Vector3D.as

```diff
@@ -0,0 +1,146 @@
+package flash.geom {
+    public class Vector3D {
+
+        // TODO: does the order matter?
```

Comment:
Order does matter. The trivial example is `describeType` that dumps all fields of a given class/object, in the order they're defined on.

Reply:
Okay, I'll take a look at `describeType`!

Reply:
Do the static members and all the methods have to be in the right order as well?

Reply:
I don't think we need to worry about exactly matching output of `describeType`. We already don't match the ordering or property/field distinction in many many places; I feel like this qualifies as extremely low concern, at least compared to many many other compatibility issues.
And if some code was doing enough black magic to use this XML output in some way, it hopefully should also have been smart enough to not rely on exact ordering :) Otherwise it might as well have hardcoded the field names instead of using `describeType` after all.

Reply:
(In particular, in FP the `getslot` and `callmethod` APIs (that rely on exact ID assignment of fields/methods, which relies on their order) were explicitly weakened/disabled to prevent one from trying to rely on things like ordering in builtin classes.)

Reply:
Okay, got it!